src/Utilities/StaticCache.hpp

```diff
@@ -27,13 +27,14 @@ struct CacheRange {
 /// \ingroup UtilitiesGroup
 /// Range of integral values for StaticCache indices.  The `Start` is inclusive
 /// and the `End` is exclusive.  The range must not be empty.
-template <typename IntegralType, IntegralType Start, IntegralType End>
+template <auto Start, auto End>
```

Comment:
you should be able to now combine IntegralCacheRange and CacheRange

Reply:
this involved putting `_st` literal suffixes in a lot of places that otherwise passed `int` into the argument, so this commit changed a bit.